src/lib/containers/SearchV2.tsx

```diff
@@ -123,11 +123,12 @@ class Search extends React.Component<InternalSearchProps, SearchState> {
     event.preventDefault()
     const { searchText } = this.state
     let { columnName } = this.state
+    const { searchable } = this.props
     if (columnName === '') {
       // default to the first one, will always be defined
       columnName =
         this.props.data?.columnModels?.filter(el =>
-          this.isSupportedColumnAndInProps(el),
```

Comment:
This was a bug that I introduced.  Now isSupportedColumnAndInProps means include it if it's "searchable" (listed in the props as a search column), or (if no search columns were defined) if it supports search.  isSupportedColumn() now only tells you if it's a column type that supports search.

Reply:
Got it.